sdk/servicebus/Microsoft.Azure.ServiceBus/src/Microsoft.Azure.ServiceBus.csproj

```diff
@@ -2,8 +2,7 @@
   <PropertyGroup>
     <AssemblyTitle>Azure ServiceBus SDK</AssemblyTitle>
     <Description>This is the next generation Azure Service Bus .NET Standard client library that focuses on queues &amp; topics. For more information about Service Bus, see https://azure.microsoft.com/en-us/services/service-bus/</Description>
-    <Version>5.2.0-beta.1</Version>
-    <ApiCompatVersion>5.1.1</ApiCompatVersion>
```

Comment:
Are we running API Compat on T1 libraries?   If so, this should stay.     

//cc: @JoshLove-msft, do you know for sure?

Reply:
I'm not sure, but this would get added automatically after release to reference the new version.

Reply:
But ideally, we wouldn't touch the ApiCompatVersion as it should be handled automatically.

Reply:
Please revert the ApiCompatVersion deletion and then we should be good to merge.

Reply:
/cc @weshaggard do you know if ApiCompat is run for T1 libraries?

Reply:
https://github.com/Azure/azure-sdk-for-net/issues/18995

Reply:
Yes ApiCompat should be run on any library that is participating in our engineering system.